packages/web-components/fast-foundation/src/tree-item/tree-item.ts

```diff
@@ -214,11 +208,11 @@ export class TreeItem extends FASTElement {
         return treeChildren ? treeChildren.length : 0;
     }
 
-    public isNestedItem(): null | boolean {
-        return (
+    public readonly isNestedItem = (): boolean => {
+        return !!(
```

Comment:
Can this be made a little clearer? Would this work?

```suggestion
        return isTreeItemElement(this.parentElement as HTMLElement);
```

Reply:
For the most part - may still need to ensure boolean, but updating to use isTreeEl would be a good use of existing code. I'll update